Source/Core/FeatureDetection.js

```diff
@@ -304,6 +308,7 @@ var FeatureDetection = {
   supportsPointerEvents: supportsPointerEvents,
   supportsImageRenderingPixelated: supportsImageRenderingPixelated,
   supportsWebP: supportsWebP,
+  supportsBasis: supportsBasis,
```

Comment:
I had to move the definition of `FeatureDetection.supportsBasis` a bit because of the circular dependency between `supportsBasis()` and `FeatureDetection.supportsWebAssembly()`.